src/MapperContext.php

```diff
@@ -246,4 +247,19 @@ public static function withNewContext(array $context, string $attribute): array
 
         return $context;
     }
+
+    public static function getForcedTimezone(array $context): ?\DateTimeZone
```

Comment:
I think you should put:
```suggestion
 /**
     * @param array{datetime_force_timezone?: string} $context
     */
    public static function getForcedTimezone(array $context): ?\DateTimeZone
```